examples/plugins/src/package-json/src/integration/utils.ts

```diff
@@ -0,0 +1,106 @@
+import { AuditOutput, Issue } from '@code-pushup/models';
+import {
+  factorOf,
+  findLineNumberInText,
+  pluralizeToken,
+} from '../../../../../../dist/packages/utils';
+import { PackageJson, SourceResult } from './types';
+
+export function baseAuditOutput(slug: string): AuditOutput {
+  return {
+    slug,
+    score: 1,
+    value: 0,
+    displayValue: pluralizePackage(),
+  } satisfies AuditOutput;
+}
+
+export function filterSeverityError(issue: Issue): issue is Issue {
```

Comment:
This type predicate doesn't narrow anything:

```suggestion
export function filterSeverityError(issue: Issue): boolean {
```